src/test/java/solvd/laba/ermakovich/hu/service/BaseTest.java

```diff
@@ -0,0 +1,48 @@
+package solvd.laba.ermakovich.hu.service;
```

Comment:
@kermakovich this is not the test, related to your main source code logic

since then, move this class to some other place